scripts/utils.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+die() { set +v; echo "$*" 1>&2 ; exit 1; }
+
+add_CLI_ARGS() {
+    # Helper for process_cells to build argument list.
+
+    FILE_TYPE=$1
+    DATA_TITLE=$2
+
+    FILE="$OUTPUT/$DATA_TITLE.$FILE_TYPE.json"
+    if [ -e "$FILE" ]
+    then
+        echo "$FILE_TYPE output already exists: $FILE"
+    else
+        CLI_ARGS="$CLI_ARGS --${FILE_TYPE}_file $FILE"
+    fi
+}
+
+usage() {
+    echo "$0 processes and prepares data for visualization using Vitessce."
+    echo "Usage: $0 --b <directory> --i <directory> --o <directory> --t <target>"
+    echo "\t -b \t Base directory"
+    echo "\t -i \t Input directory"
+    echo "\t -o \t Output directory"
+    echo "\t -t \t Amazon S3 target\n"
```

Comment:
- Not necessary here, but you could use `basename "$0"`: Depending on where it was called from, there could be directories you don't need included in the message. 
- First line has `--`; later lines `-`.
- Where messages are just supposed to be human readable, I like to make sure they come out on STDERR. The `1>&2` in `die` above is an example. (Same idea applies to python scripts). The best example is where you're pipelining commands together, and the first command has an error: If it's on STDOUT you'll never see it.
- Consider a heredoc or multiline string: Particularly with the addition of `1>&2` on each line, it will start to get cluttered.